src/Sylius/Component/Resource/Model/ToggleableTrait.php

```diff
@@ -22,6 +23,10 @@
     protected $enabled = true;
 
     /**
+     * @var ArrayCollection
+     */
+    protected $coll;
+    /**
```

Comment:
I would say that these changes should be reverted. That was probably committed by accident.